tests/core/conversion/converters/test_batch_norm.cpp

```diff
@@ -36,6 +36,39 @@ TEST(Converters, ATenBatchNormConvertsCorrectly) {
       torch_tensorrt::tests::util::almostEqual(jit_results[0], trt_results[0].reshape_as(jit_results[0]), 2e-6));
 }
 
+TEST(Converters, ATenBatchNormAffineFalseConvertsCorrectly) {
+  // BatchNorm(ch, affine=False)
+  const auto graph = R"IR(
+      graph(%0 : Tensor,
+            %1: NoneType = prim::Constant(),
```

Comment:
Just fyi, the nones  in the graph inputs can be consolidated as a single `%1: NoneType = prim::Constant()` in graph so that you don't need to pass these as arguments but this should be fine